sessions/class-01/exercise/start.js

```diff
@@ -67,7 +88,13 @@ export function hasValue(item) {}
  * @param {string} name Name of the person.
  * @returns {string[]} Array where firstName is in 0th position and last name is in 1st position.
  */
-export function getFirstAndLastNames(name) {}
+export function getFirstAndLastNames(name) {
+    let allNames = name.split(' ');
+    let firstName = allNames[0];
+    allNames.splice(0,1)
+    let lastName = allNames.join(' ');
+    return [firstName, lastName];
+}
```

Comment:
Here also we shouldn't use `let` because none of the variables are reassigned. So use `const`.